src/components/Header/index.js

```diff
@@ -37,79 +60,28 @@ const Header = ({ path, blue, logoColour }) => {
-              <Flex alignCenter wrap justifyBetween>
-                <LogoLink to="/">
-                  <Logo path={path} blue={blue} logoColour={logoColour} />
-                </LogoLink>
-                <Hamburger
-                  light={!!blue}
-                  onClick={() => toggleMobileNavbar(!isMobileNavbarOpen)}
-                />
-                <Navbar
-                  isOpen={isMobileNavbarOpen}
-                  onClose={() => toggleMobileNavbar(false)}
-                >
-                  <HomeLink activeClassName="active" to="/">
-                    Home
-                  </HomeLink>
-                  <HeaderAnchor
-                    light={!!blue}
-                    activeClassName="active"
-                    to="/engineering/"
-                  >
-                    Engineering
-                  </HeaderAnchor>
-                  <HeaderAnchor
-                    light={!!blue}
-                    activeClassName="active"
-                    to="/design/"
-                  >
-                    Design
-                  </HeaderAnchor>
-                  <HeaderAnchor
-                    light={!!blue}
-                    activeClassName="active"
-                    to="/training/"
-                  >
-                    Training
-                  </HeaderAnchor>
-                  <HeaderAnchor
-                    light={!!blue}
-                    activeClassName="active"
-                    to="/our-work/"
-                  >
-                    Our work
-                  </HeaderAnchor>
-                  <HeaderAnchor
-                    light={!!blue}
-                    activeClassName="active"
-                    to="/contact/"
-                  >
-                    Contact
-                  </HeaderAnchor>
-                  <HeaderAnchor
-                    light={!!blue}
-                    href="https://medium.com/yld-engineering-blog/"
-                  >
-                    Blog
-                  </HeaderAnchor>
-                  <HeaderAnchor
-                    light={!!blue}
-                    activeClassName="active"
-                    to="/join-us/"
-                  >
-                    Join Us
-                  </HeaderAnchor>
-                </Navbar>
-                <Overlay
-                  visible={isMobileNavbarOpen}
-                  onClick={() => toggleMobileNavbar(!isMobileNavbarOpen)}
-                />
-              </Flex>
-            </Padding>
-          </header>
+          <Padding bottom={3}>
+            <FixedHeightFlex alignCenter justifyBetween as="header">
+              <Link to="/">
+                <Logo path={path} blue={blue} logoColour={logoColour} />
+              </Link>
+              <TopNavbar links={links} dark={blue} />
+              <Hamburger
+                light={!!blue}
+                onClick={() => toggleMobileNavbar(!isMobileNavbarOpen)}
+              />
+              <Overlay
```

Comment:
Probably the `Overlay` and `SideNavbar` could be encapsulated by a `MobileNavigation` or `MobileMenu` component?

Reply:
Yeah, it's a good point. Tbh I was really unsure about what to include in the definition of each `nav` anyway.

Is the TopNavbar just the set of similar-looking links? Or would you say the YLD logo is in the TopNavbar? Since there is a separate 'Home' link in the SideNav, perhaps the home navigation (YLD logo) _should_ be part of TopNav? Then what's the distinction between the `header` & `nav` in our case?

Would you say the hamburger is part of the TopNavbar or the SideNavbar? If you say that the `nav` is only the set of links, then is the blue/black background (what I'm calling the Modal) actually part of the SideNav?

Or maybe the SideNav is only the set of links, which sit inside the modal. In which case, the Overlay is no more part of the SideNav than the Modal is.

Not sure if that's particularly easy to understand, written down... but what do you think "nav" should mean?

Reply:
In a general approach, I use these rules of thumb:
- `nav` is literally a `nav` tag with a `ul` and `li`s with anchors.
- a component that has one of those `nav` inside, should be called `Navigation` (or something similar`

So, for this scenario, I'd imagine something like:
- `Header` is the top-level component that has
  - `MobileNavigation`
  - `DesktopNavigation`
  - `Hamburger`
  - `Logo(s)`

What do you think?

Reply:
Yeah I think we're on the same page really. This is what I've gone with:
> `nav` is literally a `nav` tag with a `ul` and `li`s with anchors

which is why the Hamburger is outside the TopNavbar & also why the Overlay is outside the SideNavbar.

Having done a bunch of googling before getting started, it feels like there's little to no convention out there & that everyone does whatever makes sense for their project. So I reckon it's all personal preference really. I'm happy to change it from this formulation to whatever you suggest.

Maybe it's easier to chat about this in person, when you get to London today?